core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -544,29 +544,26 @@ protected function createExtract()
 			'collate' => $sql['collate']
 		);
 
+		$this->arrFields = array();
+		$this->arrOrderFields = array();
+
 		// Fields
-		if (!empty($fields))
```

Comment:
In case you wonder: the if condition can be removed, because there is an `empty($fields)` check in line 516 already.

Reply:
PhpStorm 2020.2 FTW :D

Reply:
Exactly. 😄